zulip_bots/zulip_bots/bots/joinme/doc.md

```diff
@@ -0,0 +1 @@
+  Zulip bot that will post a link to Joinme call.
```

Comment:
You are going to have to elaborate on how the bot works a bit more. See [the giphy bot's doc.md](https://github.com/zulip/python-zulip-api/blob/master/zulip_bots/zulip_bots/bots/giphy/doc.md) for an example of what we are looking for.